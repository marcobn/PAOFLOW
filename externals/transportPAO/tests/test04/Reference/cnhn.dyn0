   6   1   1
   4
   0.000000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
   0.166666666666667E+00   0.000000000000000E+00   0.000000000000000E+00
   0.333333333333333E+00   0.000000000000000E+00   0.000000000000000E+00
  -0.500000000000000E+00   0.000000000000000E+00   0.000000000000000E+00
