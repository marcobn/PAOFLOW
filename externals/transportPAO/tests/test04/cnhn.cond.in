
  &INPUT_CONDUCTOR
    postfix='_CnHn'
    work_dir = './'
    dimC = 48
    calculation_type = 'bulk'
    carriers= 'phonons'
    ne = 3000
    emin = 0.0
    emax =  3100.0
    datafile_C = "./ext.ham"
    transport_dir = 1
    delta = 1.0d-8
  /

<HAMILTONIAN_DATA>

   <H00_C  cols="ALL" rows="ALL" />
   <H_CR   cols="ALL" rows="ALL" />

</HAMILTONIAN_DATA>
