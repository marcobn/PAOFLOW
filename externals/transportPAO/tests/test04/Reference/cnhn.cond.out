
  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <conductor>  v. 2.5.0alpha1  starts ...
  Date  1Aug2012 at 15:44:41 

     Serial run.



          BUILT :    Mon Jul 30 11:07:54 2012
           HOST :    i686-apple-darwin11.4.0
           ARCH :    mac686
             CC :    cc
            CPP :    cpp
            F90 :    gfortran
            F77 :    gfortran
         DFLAGS :    -D__GFORTRAN -D__STD_F95 -D__FFTW
      BLAS LIBS :    -L/Users/marco/Programs/trunk/want/extlibs/blas/lib -lblas
    LAPACK LIBS :    -L/Users/marco/Programs/trunk/want/extlibs/lapack/lib -llapack
       FFT LIBS :    
      MASS LIBS :    



  ======================================================================
  =  Conductor Init                                                    =
  ======================================================================

  ./ext.ham file fmt: internal

  ======================================================================
  =  INPUT Summary                                                     =
  ======================================================================

  <INPUT>
          Calculation Type :     bulk
                    prefix :     
                   postfix :     _CnHn
                  work_dir :     ./
               L-lead dim. :        48
            conductor dim. :        48
               R-lead dim. :        48
       Conductance Formula :     landauer
                  Carriers :     phonons
       Transport Direction :         1
          Have Correlation :     FALSE
              Write k-data :     FALSE
            Write sgm lead :     FALSE
                Write gf C :     FALSE
           Max iter number :       200
                    nprint :        20

        Conductor datafile :     ./ext.ham
       leads are identical :     TRUE 
         ovp orthogonaliz. :     FALSE
  </INPUT>


  <ENERGY_GRID>
                 Dimension :       3000
                 Buffering :          1
             Min Frequency :        0.00000
             Max Frequency :     **********
               Energy Step :     3204.40147
                     Delta :        0.00000
             Smearing Type :     lorentzian
             Smearing grid :      20000
             Smearing gmax :       25.00000
                   Shift_L :        0.00000
                   Shift_C :        0.00000
                   Shift_R :        0.00000
                Shift_corr :        0.00000
  </ENERGY_GRID>


  <K-POINTS>
       nkpts_par =    1
       nrtot_par =    1
        use_symm = TRUE

       Parallel kpoints grid:        nk = (  1  1  1 )   s = (  0  0  0 )
       k (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000

       Parallel R vector grid:       nr = (  1  1  1 )
       R (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000
  </K-POINTS>


  <PARALLELISM>
       Paralellization over frequencies
       # of processes:     1
  </PARALLELISM>


 <MEMORY_USAGE>
                smearing:          0.480 MB
                 kpoints:          0.000 MB
        hamiltonian data:          0.555 MB
        correlation data:          0.002 MB
               workspace:          0.000 MB

     Total alloc. Memory:          1.037 MB
 </MEMORY_USAGE>


  ======================================================================
  =  Frequency Loop                                                    =
  ======================================================================

  Computing omega(     1 ) =      5.66074 cm-1
  T matrix converged after avg. # of iterations      9.500

                              Total time spent up to now :     0.19 secs

  Computing omega(    20 ) =    246.74608 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :     1.29 secs

  Computing omega(    40 ) =    353.51331 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :     2.52 secs

  Computing omega(    60 ) =    434.80994 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :     3.78 secs

  Computing omega(    80 ) =    503.13787 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :     5.04 secs

  Computing omega(   100 ) =    563.23685 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :     6.30 secs

  Computing omega(   120 ) =    617.51419 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :     7.54 secs

  Computing omega(   140 ) =    667.39179 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :     8.27 secs

  Computing omega(   160 ) =    713.79257 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :     8.51 secs

  Computing omega(   180 ) =    757.35584 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :     8.73 secs

  Computing omega(   200 ) =    798.54611 cm-1
  T matrix converged after avg. # of iterations      8.500

                              Total time spent up to now :     9.01 secs

  Computing omega(   220 ) =    837.71351 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    10.18 secs

  Computing omega(   240 ) =    875.12968 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    11.34 secs

  Computing omega(   260 ) =    911.01042 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    12.52 secs

  Computing omega(   280 ) =    945.53054 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    13.72 secs

  Computing omega(   300 ) =    978.83402 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    14.91 secs

  Computing omega(   320 ) =   1011.04108 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :    15.85 secs

  Computing omega(   340 ) =   1042.25337 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    16.04 secs

  Computing omega(   360 ) =   1072.55775 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    16.23 secs

  Computing omega(   380 ) =   1102.02911 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :    16.47 secs

  Computing omega(   400 ) =   1130.73259 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    17.01 secs

  Computing omega(   420 ) =   1158.72525 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    18.23 secs

  Computing omega(   440 ) =   1186.05744 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    19.45 secs

  Computing omega(   460 ) =   1212.77379 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    20.66 secs

  Computing omega(   480 ) =   1238.91416 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    21.85 secs

  Computing omega(   500 ) =   1264.51427 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    23.08 secs

  Computing omega(   520 ) =   1289.60628 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    24.30 secs

  Computing omega(   540 ) =   1314.21931 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    25.54 secs

  Computing omega(   560 ) =   1338.37977 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    26.81 secs

  Computing omega(   580 ) =   1362.11176 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    28.09 secs

  Computing omega(   600 ) =   1385.43729 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    29.35 secs

  Computing omega(   620 ) =   1408.37655 cm-1
  T matrix converged after avg. # of iterations      1.500

                              Total time spent up to now :    29.93 secs

  Computing omega(   640 ) =   1430.94813 cm-1
  T matrix converged after avg. # of iterations      1.500

                              Total time spent up to now :    30.24 secs

  Computing omega(   660 ) =   1453.16915 cm-1
  T matrix converged after avg. # of iterations      1.500

                              Total time spent up to now :    30.52 secs

  Computing omega(   680 ) =   1475.05546 cm-1
  T matrix converged after avg. # of iterations      1.500

                              Total time spent up to now :    30.81 secs

  Computing omega(   700 ) =   1496.62174 cm-1
  T matrix converged after avg. # of iterations      2.000

                              Total time spent up to now :    31.11 secs

  Computing omega(   720 ) =   1517.88163 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    32.20 secs

  Computing omega(   740 ) =   1538.84784 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    33.46 secs

  Computing omega(   760 ) =   1559.53221 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    34.73 secs

  Computing omega(   780 ) =   1579.94580 cm-1
  T matrix converged after avg. # of iterations     11.000

                              Total time spent up to now :    35.98 secs

  Computing omega(   800 ) =   1600.09899 cm-1
  T matrix converged after avg. # of iterations     10.500

                              Total time spent up to now :    37.27 secs

  Computing omega(   820 ) =   1620.00148 cm-1
  T matrix converged after avg. # of iterations      1.500

                              Total time spent up to now :    38.28 secs

  Computing omega(   840 ) =   1639.66241 cm-1
  T matrix converged after avg. # of iterations      1.000

                              Total time spent up to now :    38.52 secs

  Computing omega(   860 ) =   1659.09037 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    38.72 secs

  Computing omega(   880 ) =   1678.29345 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    38.92 secs

  Computing omega(   900 ) =   1697.27927 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    39.12 secs

  Computing omega(   920 ) =   1716.05505 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    39.32 secs

  Computing omega(   940 ) =   1734.62762 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    39.49 secs

  Computing omega(   960 ) =   1753.00342 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    39.67 secs

  Computing omega(   980 ) =   1771.18859 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    39.85 secs

  Computing omega(  1000 ) =   1789.18894 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.05 secs

  Computing omega(  1020 ) =   1807.00999 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.23 secs

  Computing omega(  1040 ) =   1824.65699 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.41 secs

  Computing omega(  1060 ) =   1842.13494 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.59 secs

  Computing omega(  1080 ) =   1859.44862 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.76 secs

  Computing omega(  1100 ) =   1876.60257 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    40.94 secs

  Computing omega(  1120 ) =   1893.60113 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.14 secs

  Computing omega(  1140 ) =   1910.44845 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.31 secs

  Computing omega(  1160 ) =   1927.14849 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.50 secs

  Computing omega(  1180 ) =   1943.70505 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.66 secs

  Computing omega(  1200 ) =   1960.12177 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.79 secs

  Computing omega(  1220 ) =   1976.40213 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    41.92 secs

  Computing omega(  1240 ) =   1992.54948 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.07 secs

  Computing omega(  1260 ) =   2008.56701 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.20 secs

  Computing omega(  1280 ) =   2024.45782 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.33 secs

  Computing omega(  1300 ) =   2040.22487 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.46 secs

  Computing omega(  1320 ) =   2055.87099 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.60 secs

  Computing omega(  1340 ) =   2071.39894 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.73 secs

  Computing omega(  1360 ) =   2086.81135 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.86 secs

  Computing omega(  1380 ) =   2102.11075 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    42.99 secs

  Computing omega(  1400 ) =   2117.29961 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.13 secs

  Computing omega(  1420 ) =   2132.38029 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.26 secs

  Computing omega(  1440 ) =   2147.35505 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.40 secs

  Computing omega(  1460 ) =   2162.22611 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.53 secs

  Computing omega(  1480 ) =   2176.99558 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.66 secs

  Computing omega(  1500 ) =   2191.66553 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.79 secs

  Computing omega(  1520 ) =   2206.23794 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    43.92 secs

  Computing omega(  1540 ) =   2220.71472 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.07 secs

  Computing omega(  1560 ) =   2235.09774 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.19 secs

  Computing omega(  1580 ) =   2249.38879 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.33 secs

  Computing omega(  1600 ) =   2263.58962 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.46 secs

  Computing omega(  1620 ) =   2277.70191 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.58 secs

  Computing omega(  1640 ) =   2291.72730 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.71 secs

  Computing omega(  1660 ) =   2305.66737 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.84 secs

  Computing omega(  1680 ) =   2319.52367 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    44.97 secs

  Computing omega(  1700 ) =   2333.29769 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.11 secs

  Computing omega(  1720 ) =   2346.99087 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.24 secs

  Computing omega(  1740 ) =   2360.60462 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.37 secs

  Computing omega(  1760 ) =   2374.14030 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.50 secs

  Computing omega(  1780 ) =   2387.59926 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.63 secs

  Computing omega(  1800 ) =   2400.98277 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.76 secs

  Computing omega(  1820 ) =   2414.29208 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    45.89 secs

  Computing omega(  1840 ) =   2427.52843 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.03 secs

  Computing omega(  1860 ) =   2440.69300 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.17 secs

  Computing omega(  1880 ) =   2453.78694 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.31 secs

  Computing omega(  1900 ) =   2466.81138 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.43 secs

  Computing omega(  1920 ) =   2479.76741 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.56 secs

  Computing omega(  1940 ) =   2492.65610 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.70 secs

  Computing omega(  1960 ) =   2505.47849 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.84 secs

  Computing omega(  1980 ) =   2518.23559 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    46.97 secs

  Computing omega(  2000 ) =   2530.92839 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.12 secs

  Computing omega(  2020 ) =   2543.55786 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.25 secs

  Computing omega(  2040 ) =   2556.12492 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.38 secs

  Computing omega(  2060 ) =   2568.63050 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.50 secs

  Computing omega(  2080 ) =   2581.07548 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.63 secs

  Computing omega(  2100 ) =   2593.46075 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.76 secs

  Computing omega(  2120 ) =   2605.78716 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    47.89 secs

  Computing omega(  2140 ) =   2618.05553 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.02 secs

  Computing omega(  2160 ) =   2630.26667 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.16 secs

  Computing omega(  2180 ) =   2642.42139 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.29 secs

  Computing omega(  2200 ) =   2654.52045 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.42 secs

  Computing omega(  2220 ) =   2666.56462 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.55 secs

  Computing omega(  2240 ) =   2678.55463 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.69 secs

  Computing omega(  2260 ) =   2690.49120 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.81 secs

  Computing omega(  2280 ) =   2702.37506 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    48.94 secs

  Computing omega(  2300 ) =   2714.20688 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.09 secs

  Computing omega(  2320 ) =   2725.98734 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.21 secs

  Computing omega(  2340 ) =   2737.71712 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.35 secs

  Computing omega(  2360 ) =   2749.39685 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.48 secs

  Computing omega(  2380 ) =   2761.02718 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.60 secs

  Computing omega(  2400 ) =   2772.60872 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.73 secs

  Computing omega(  2420 ) =   2784.14208 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.86 secs

  Computing omega(  2440 ) =   2795.62787 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    49.99 secs

  Computing omega(  2460 ) =   2807.06666 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.14 secs

  Computing omega(  2480 ) =   2818.45902 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.27 secs

  Computing omega(  2500 ) =   2829.80552 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.40 secs

  Computing omega(  2520 ) =   2841.10670 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.53 secs

  Computing omega(  2540 ) =   2852.36311 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.66 secs

  Computing omega(  2560 ) =   2863.57527 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.81 secs

  Computing omega(  2580 ) =   2874.74371 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    50.95 secs

  Computing omega(  2600 ) =   2885.86892 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.09 secs

  Computing omega(  2620 ) =   2896.95140 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.22 secs

  Computing omega(  2640 ) =   2907.99166 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.35 secs

  Computing omega(  2660 ) =   2918.99015 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.48 secs

  Computing omega(  2680 ) =   2929.94736 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.60 secs

  Computing omega(  2700 ) =   2940.86374 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.73 secs

  Computing omega(  2720 ) =   2951.73976 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    51.89 secs

  Computing omega(  2740 ) =   2962.57584 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.01 secs

  Computing omega(  2760 ) =   2973.37244 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.15 secs

  Computing omega(  2780 ) =   2984.12997 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.33 secs

  Computing omega(  2800 ) =   2994.84886 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.52 secs

  Computing omega(  2820 ) =   3005.52953 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.74 secs

  Computing omega(  2840 ) =   3016.17237 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    52.92 secs

  Computing omega(  2860 ) =   3026.77779 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    53.12 secs

  Computing omega(  2880 ) =   3037.34618 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    53.66 secs

  Computing omega(  2900 ) =   3047.87793 cm-1
  T matrix converged after avg. # of iterations     10.000

                              Total time spent up to now :    54.82 secs

  Computing omega(  2920 ) =   3058.37340 cm-1
  T matrix converged after avg. # of iterations      9.500

                              Total time spent up to now :    55.95 secs

  Computing omega(  2940 ) =   3068.83299 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    56.52 secs

  Computing omega(  2960 ) =   3079.25704 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    56.70 secs

  Computing omega(  2980 ) =   3089.64593 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    56.91 secs

  Computing omega(  3000 ) =   3100.00000 cm-1
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :    57.10 secs


  ======================================================================
  =  Writing data                                                      =
  ======================================================================


  conductance written on file:    cond_CnHn.dat

  doscond written on file:    doscond_CnHn.dat

  ======================================================================

   <global routines>
             clock number :    13

                conductor :    57.00s CPU 

           datafiles_init :     0.02s CPU
            smearing_init :     0.02s CPU
                   cft_1z :     0.01s CPU (       2 calls,   0.007 s avg)
         hamiltonian_init :     0.06s CPU
              read_matrix :     0.06s CPU (       2 calls,   0.030 s avg)
              fourier_par :     0.00s CPU (       4 calls,   0.000 s avg)
             do_conductor :    56.99s CPU
        hamiltonian_setup :     0.13s CPU (    3000 calls,   0.000 s avg)
            transfer_mtrx :    46.52s CPU (    3000 calls,   0.016 s avg)
                    green :     4.01s CPU (    6000 calls,   0.001 s avg)
              gzero_maker :     0.18s CPU (    9000 calls,   0.000 s avg)
            transmittance :     2.38s CPU (    3000 calls,   0.001 s avg)


